internal-packages/db/src/utils/rate-limit-utils.ts

```
@@ -0,0 +1,108 @@
+import { Plan } from "../types";
+
+function nextReset(now: Date, interval: 'hour' | 'month'): Date {
+  const next = new Date(now);
+  switch (interval) {
+    case 'hour':
+      next.setHours(next.getHours() + 1, 0, 0, 0);
+      break;
+    case 'month':
+      next.setDate(1);
+      const newMonth = next.getMonth() + 1;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)